Orange/widgets/data/owpurgedomain.py

```diff
@@ -94,8 +94,7 @@ def __init__(self):
                 gui.separator(box3, 2)
             gui.label(box3, self, "{}: %({})s".format(label, value))
 
-        gui.auto_commit(self.buttonsArea, self, "autoSend", "Send Data",
-                        checkbox_label="Send automatically",
+        gui.auto_commit(self.buttonsArea, self, "autoSend", "Apply",
```

Comment:
Also here: instead of "Send automatically" we now have "Auto Send Is On", which is uglier.